pkg/policy/api/rule.go

```diff
@@ -216,6 +216,29 @@ type K8sServiceSelectorNamespace struct {
 //   if if multiple members of the structure are specified, then all members
 //   must match in order for the rule to take effect.
 type EgressRule struct {
+	// ToEndpoints is a list of endpoints identified by an EndpointSelector to
+	// which the endpoints subject to the rule are allowed to communicate.
+	//
+	// Example:
+	// Any endpoint with the label "role=backend" can communicate with any
```

Comment:
nit: It will sound more logical to use the example of allowing frontend to backend